Item/Item.module.less

```diff
@@ -0,0 +1,30 @@
+// Item.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.item {
+	position: relative;
```

Comment:
Why is this not in `.applySkins()`?

Reply:
Practically speaking, it's not a variable defined in a `colors-*` or `variables-*` LESS file, so it doesn't _need_ to be. Also, having it here means it has lower precedence and is, therefore, easier to override from a user perspective.

Alternatively, we have other code that fits this same rule which we left in the `applySkins` block for our (developer) convenience. Consistency is probably the winner of this though.

Reply:
I'd agree that consistency is important for this code base. If you think this warrants an exception, I'd suggest documenting it inline so the author understands the rationale.

Reply:
I made the rest of the files match this approach, so it's a bit more apparent what's going on.